express.cjs

```diff
@@ -0,0 +1,25 @@
+const express = require('express')
```

Comment:
File name can stay the same as server.cjs. Don't need ti update it based on the framework used.